Typescript errors in generator.

# What happened
I am generating an application on the new Angular front-end framework. Generation of the back-end fails with this message (a log fragment):
```
=> CACHED [prototype 1/8] FROM docker.io/ampersandtarski/prototype-framework:main@sha256:82cf23df696f3cac43f3b95678fdea2c36fdcd8c6c4a89ff4c74ca20f437bf9a                                                                                                                                                                                                                                                                             0.0s
 => [prototype 2/8] COPY . /usr/local/project/                                                                                                                                                                                                                                                                                                                                                                                         0.1s
 => [prototype 3/8] RUN ampersand proto --no-frontend /usr/local/project/main.adl   --proto-dir /var/www/backend   --verbose                                                                                                                                                                                                                                                                                                          58.5s
 => [prototype 4/8] RUN ampersand proto --frontend-version Angular --no-backend /usr/local/project/main.adl   --proto-dir /var/www/frontend/src/app/generated   --verbose                                                                                                                                                                                                                                                             57.9s
 => [prototype 5/8] COPY customizations /var/www/                                                                                                                                                                                                                                                                                                                                                                                      0.0s 
 => [prototype 6/8] WORKDIR /var/www/frontend                                                                                                                                                                                                                                                                                                                                                                                          0.0s 
 => ERROR [prototype 7/8] RUN npx ng build                                                                                                                                                                                                                                                                                                                                                                                            92.1s 
------                                                                                                                                                                                                                                                                                                                                                                                                                                      
 > [prototype 7/8] RUN npx ng build:                                                                                                                                                                                                                                                                                                                                                                                                        
4.082 - Generating browser application bundles (phase: setup)...                                                                                                                                                                                                                                                                                                                                                                            
90.16 ✔ Browser application bundle generation complete.                                                                                                                                                                                                                                                                                                                                                                                     
91.75 ✔ Browser application bundle generation complete.                                                                                                                                                                                                                                                                                                                                                                                     
91.76                                                                                                                                                                                                                                                                                                                                                                                                                                       
91.76 Error: src/app/generated/project.views.ts:9:4 - error TS1138: Parameter declaration expected.
91.76 
91.76 9   (: '(';
91.76      ~
91.76 
91.76 
91.76 Error: src/app/generated/project.views.ts:9:5 - error TS7006: Parameter '(Missing)' implicitly has an 'any' type.
91.76 
91.76 9   (: '(';
91.76       
91.76 
91.76 
91.76 Error: src/app/generated/project.views.ts:9:6 - error TS1003: Identifier expected.
91.76 
91.76 9   (: '(';
91.76        ~~~
91.76 
91.76 
91.76 Error: src/app/generated/project.views.ts:9:9 - error TS1138: Parameter declaration expected.
91.76 
91.76 9   (: '(';
91.76           ~

```
# What I expected

# Version of ampersand that was used

# Steps to reproduce

1.
2.
3.
4.

# Screenshot / Video

# Context / Source of Dockerfile
```Dockerfile
FROM ampersandtarski/prototype-framework:main

COPY . /usr/local/project/

# Run ampersand compiler to generated backend json model files (in generics folder)
RUN ampersand proto --no-frontend /usr/local/project/main.adl \
  --proto-dir /var/www/backend \
  --verbose

# Run ampersand compiler to generated new frontend
RUN ampersand proto --frontend-version Angular --no-backend /usr/local/project/main.adl \
  --proto-dir /var/www/frontend/src/app/generated \
  --verbose

## At this place you can copy any frontend customizations (if applicable)
## Do this before running `npx ng build` below
## You can also copy here any project specific backend code

COPY customizations /var/www/

WORKDIR /var/www/frontend

# Build + bundle Angular frontend
RUN npx ng build

# Copy Angular frontend to public folder in web server
RUN cp -r /var/www/frontend/dist/prototype-frontend/* /var/www/html/
```

--- a/changelog.md
+++ b/changelog.md
@@ -17,6 +17,7 @@ Additional labels for pre-release and build metadata are available as extensions
 * [Issue 161](https://github.com/AmpersandTarski/prototype/issues/161) Bugfix ConceptNotDefinedException
 * [Issue 210](https://github.com/AmpersandTarski/prototype/issues/210) Bugfix using TableType enum for method to empty a relation population
 * [Issue 218](https://github.com/AmpersandTarski/prototype/issues/218) Improve error reporting for exceptions in excel importer
+* [Issue 221](https://github.com/AmpersandTarski/prototype/issues/221) Bugfix typescript type object property naming by surrounding view labels with quotes
 
 ## v1.18.0 (27 april 2024)
 * [Issue 79](https://github.com/AmpersandTarski/prototype/issues/79) Add support for delimited multi value columns in excel importer

--- a/frontend/src/app/generated/.templates/project.views.ts.txt
+++ b/frontend/src/app/generated/.templates/project.views.ts.txt
@@ -6,7 +6,7 @@
  */
 
 $views:{view|export type $view.viewIdTmp$View = {$view.viewSegments:{segment|
-  $segment.segmentLabel$: $segment.segmentTypescriptType$;}$$if(!view.viewIsEmpty)$
+  "$segment.segmentLabel$": $segment.segmentTypescriptType$;}$$if(!view.viewIsEmpty)$
 
 $endif$\};
 }$

